vocabularies/voc4cat/0007131.ttl

```diff
@@ -0,0 +1,37 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007131>
+    a skos:Collection ;
+    dcterms:identifier "voc4cat_0007131"^^xsd:token ;
+    dcterms:isPartOf <https://w3id.org/nfdi4cat/voc4cat> ;
+    rdfs:isDefinedBy <https://w3id.org/nfdi4cat/voc4cat> ;
+    skos:definition "A collection of terms used in various measurement techniques, relevant in the domain of catalysis. (This collection is not exclusive"@en ;
```

Comment:
There is a missing closing bracket at the end. You could also remove the "(This collection ....)" part. It does not add much info.